tutorials/plugins/ios/index.rst

```diff
@@ -0,0 +1,8 @@
+iOS plugins
+===============
```

Comment:
The number of characters below a heading *must* match the heading above:

```suggestion
iOS plugins
===========
```

This is a Sphinx/reStructuredText requirement. Make sure to do this for all the headings you've added.